tests/tests/proposal_tests.cpp

```diff
@@ -442,7 +449,7 @@ BOOST_AUTO_TEST_CASE(update_proposal) { try {
     push_tx_with_ops(tx, alice_private_key, uop11);
     generate_blocks(1);
 
-    BOOST_REQUIRE_THROW(db->get_proposal(cop.author, cop.title), fc::exception);
+    GOLOS_CHECK_OBJECT_MISSING((*db), proposal, cop.author, cop.title);
```

Comment:
Why do you use GOLOS_CHECK_OBJECT_MISSING? It's production macros and it's throw object_already_exists if proposal will be exists. This exception doesn't catch and test failed with "unknown error" message.
May be better to use: `BOOST_CHECK(nullptr == db->find_proposal(cop.author, cop.title);`?

Reply:
You're right, it's totally wrong macro. Fixed